src/loading.jl

```diff
@@ -0,0 +1,90 @@
+loadleaf!(dst, src, err) = dst
+function loadleaf!(dst::AbstractArray, src::Bool, err)
+  if iszero(src)
+    dst .= src
+  else
+    error("Cannot copy boolean parameter == true to non-zero parameter.")
+  end
+  return dst
+end
+loadleaf!(dst::Bool, src::AbstractArray, err) = iszero(dst) ? dst :
+  error("Cannot copy non-zero parameter to boolean parameter == true.")
+function loadleaf!(dst::AbstractArray, src::AbstractArray, err)
+  (size(dst) == size(src)) || throw(err)
+  copyto!(dst, src)
+end
+
+_tie_check(dst::Bool, src::AbstractArray) = iszero(dst) ||
+  error("Encountered tied parameter with boolean source at some nodes and non-boolean sources at others.")
+_tie_check(dst::AbstractArray, src::Bool) = (iszero(dst) && iszero(src)) ||
+  error("Encountered tied parameter with boolean source at some nodes and non-boolean sources at others.")
+_tie_check(dst::AbstractArray, src::AbstractArray) = (dst == src) ||
+  error("Encountered tied destination parameters with untied and mismatched sources.")
+_tie_check(dst, src) = true
+
+_bool_tie_check(dst, src) = true
+
+"""
+    loadmodel!(dst, src)
+
+Copy all the parameters (trainable and non-trainable) from `src` to `dst`.
+
+`loadmodel!` recursively walks the [`Functors.children`](@ref) of `dst` and `src`
+calling `copyto!` on any pair of children where [`Functors.isleaf`](@ref) is true.
+It also handles "absent" parameters such as `bias == false`.
+It throws an error whenever:
```

Comment:
```suggestion
Copy all the parameters (trainable and non-trainable) from `src` into `dst`.

Recursively walks `dst` and `src` together using [`Functors.children`](@ref),
and calling `copyto!` on parameter arrays.
Non-array elements (such as activation functions) need not match.
An all-zero bias array can be copied to or from absent bias, encoded `bias = false`.

Throws an error when:
```
It's inaccurate to say it calls `copyto!` on all leaf nodes. Stressing "parameter arrays" so that e.g. dropout `p` is excluded.

I think we can say exactly how it "handles" false, and should avoid "such as" when there's exactly one such case.